### PR TITLE
fix scripts, content of binary distribution zip

### PR DESCRIPTION
**Pull request by lukasj**
Signed-off-by: Lukas Jungmann <lukas.jungmann@oracle.com>

### CODE REVIEW

File: bundles/distribution/src/main/assembly/eclipselink.zip.xml [hunk 1]
```diff
@@ -24,57 +24,37 @@
...
-                <include>${project.parent.groupId}:eclipselink:*:sources</include>
-            </includes>
-        </dependencySet>
-        <dependencySet>
-            <outputDirectory>eclipselink/</outputDirectory>
-            <outputFileNameMapping>${artifact.artifactId}-javadocs.${artifact.extension}</outputFileNameMapping>
-            <includes>
-                <include>${project.parent.groupId}:eclipselink:*:javadoc</include>
-            </includes>
-        </dependencySet>
```
Comment (rfelcman) [inline bundles/distribution/src/main/assembly/eclipselink.zip.xml:45]: See javadoc comment above.

**Comment by rfelcman** [comment]
About zip bundle: could You please modify _packageRename.sh_ script too? There are should be
``. `dirname $0`/../../bin/setenv.sh`` at line 14 and
``CLASSPATH=`dirname $0`/package-rename.jar`` at line 20

File: bundles/distribution/src/main/assembly/eclipselink.zip.xml [hunk 2]
```diff
@@ -24,57 +24,37 @@
     <dependencySets>
         <dependencySet>
             <outputDirectory>eclipselink/jlib/</outputDirectory>
-            <outputFileNameMapping>${artifact.artifactId}.${artifact.extension}</outputFileNameMapping>
+            <outputFileNameMapping>${artifact.artifactId}${dashClassifier?}.${artifact.extension}</outputFileNameMapping>
             <includes>
                 <include>${project.parent.groupId}:eclipselink</include>
```
Comment (lukasj) [inline bundles/distribution/src/main/assembly/eclipselink.zip.xml:29]: @rfelcman this line here covers eclipselink.jar, eclipselink-sources.jar and eclipselink-javadoc.jar as long as they are defined as project dependencies

File: bundles/distribution/src/main/assembly/eclipselink.zip.xml [hunk 3]
```diff
@@ -24,57 +24,37 @@
...
                 <include>${project.parent.groupId}:org.eclipse.persistence.extension</include>
             </includes>
         </dependencySet>
         <dependencySet>
             <outputDirectory>eclipselink/jlib/jpa/</outputDirectory>
-            <outputFileNameMapping>${artifact.artifactId}.${artifact.extension}</outputFileNameMapping>
+            <outputFileNameMapping>${artifact.artifactId}${dashClassifier?}.${artifact.extension}</outputFileNameMapping>
             <includes>
                 <include>${project.parent.groupId}:org.eclipse.persistence.jpa.modelgen.processor</include>
                 <include>${project.parent.groupId}:org.eclipse.persistence.jpars</include>
```
Comment (lukasj) [inline bundles/distribution/src/main/assembly/eclipselink.zip.xml:44]: @rfelcman again, every classifier for given artifact available in project deps is added - so this covers jpars-javadoc and sources as well

File: bundles/distribution/pom.xml [hunk 4]
```diff
@@ -406,56 +412,6 @@
...
-                            <header><![CDATA[${javadoc.jpars.prefixTitle}]]></header>
-                            <windowtitle>${javadoc.jpars.prefixTitle}</windowtitle>
-                            <dependencySourceIncludes>
-                                <dependencySourceInclude>org.eclipse.persistence:org.eclipse.persistence.jpars</dependencySourceInclude>
-                            </dependencySourceIncludes>
-                            <finalName>eclipselink-jpars</finalName>
-                        </configuration>
-                    </execution>
-                </executions>
-            </plugin>
```
[the remark this answers is not in the document]
Comment (lukasj) [reply bundles/distribution/pom.xml:458]: it's there just being grabbed by different step. current zip actually (probably-I did no check the nosql stuff) contains bit more (extensions, sources)

File: bundles/distribution/pom.xml [hunk 5]
```diff
@@ -553,26 +509,6 @@
...
-                        </goals>
-                        <configuration>
-                            <target>
-                                <unzip src="${project.build.directory}/eclipselink.zip" dest="${eclipselink.unzip.dir}"/>
-                                <chmod dir="${eclipselink.unzip.dir}/eclipselink/bin" perm="+x" includes="**/*"/>
-                            </target>
-                        </configuration>
-                    </execution>
-                </executions>
-            </plugin>
```
[the remark this answers is not in the document]
Comment (lukasj) [reply bundles/distribution/pom.xml:575]: I moved them to default run

File: bundles/distribution/pom.xml [hunk 6]
```diff
@@ -538,41 +556,31 @@
...
+                        <configuration>
+                            <environmentVariables>
+                                <ECLIPSELINK_UNZIP_DIR>${eclipselink.unzip.dir}</ECLIPSELINK_UNZIP_DIR>
+                            </environmentVariables>
+                            <includes>
+                                <include>org.eclipse.persistence.testing.jaxb.installer.InstallerTestSuite
+                                </include>
+                            </includes>
+                        </configuration>
+                    </execution>
```
Comment (rfelcman) [inline bundles/distribution/pom.xml:578]: please add there 
```                    <execution>
                        <id>verify-integration-tests</id>
                        <goals>
                            <goal>verify</goal>
                        </goals>
                        <configuration>
                            <skip>${integration.test.skip.verify}</skip>
                        </configuration>
                    </execution>
```
to mark build result as a failure if installer-test will fail.

Comment (lukasj) [reply bundles/distribution/pom.xml:578]: done

**Pull request merged**